TabLayout/TabLayout.js

```diff
@@ -273,6 +281,24 @@ const TabLayoutBase = kind({
 				forward('onTabAnimationEnd')
 			)
 		),
+		handleFlick: ({direction, velocityX}, {collapsed, onCollapse, onExpand}) => {
+			// See the global class 'spotlight-input-touch' to check the input type is touch
+			if (isTouchMode() && direction === 'horizontal') {
+				if (!collapsed && velocityX < 0) {
+					onCollapse();
+				} else if (collapsed && velocityX > 0) {
+					onExpand();
+				}
+			}
+		},
+		handleClick: handle(
```

Comment:
What is the source of this code? If this is an additional implementation, please leave a description

Reply:
I found it from https://github.com/enactjs/sandstone/pull/828.
In the comments, PLAT-132782 is omitted.

Reply:
#828